docs/guides/Checkout/customization/displaying-sla-by-item-in-checkout.md

```diff
@@ -0,0 +1,142 @@
+
+```json
+{
+    "paymentConfiguration": {
+    ...
+    },
+    "taxConfiguration": null,
+    "minimumQuantityAccumulatedForItems": 1,
+    "decimalDigitsPrecision": 2,
+    "minimumValueAccumulated": null,
+    "apps": [
+    ...
+    ],
+    "allowMultipleDeliveries": true,
+    "allowManualPrice": null,
+    "savePersonalDataAsOptIn": false,
+    "maxNumberOfWhiteLabelSellers": null,
+    "recaptchaValidation": "vtexcriteria",
+    "recaptchaMinScore": null,
+    "recaptchaKeys": null,
+    "maskStateOnAddress": true,
+    "enableSecureCookies": true,
+    "useOwnershipCookie": true,
+    "ignoreProfileData": null,
+    "useIndividualShippingEstimates": true
+}
+```
+
+By doing so, the `shippingEstimateDate` will be updated with the SLA information configured by the sellers.
+
+## Add items to cart
+
+Once you enable the `useIndividualShippingEstimates`, you can test if the configuration is correct by adding items to a shopping cart using the [Add cart items](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/items) endpoint. See the request body example below:
+
+```json
+{
+  "orderItems": [
+    {
+      "seller": "1",
+      "id": 6,
+      "quantity": 1
+    },
+    {
+      "seller": "1",
+      "id": 11,
+      "quantity": 1
+    }
+  ]
+}
+```
+
+## Send shipping data attachment
+
+Send the item shipping data to the `orderForm` by using the [Add shipping address and select delivery option](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/attachments/shippingData) endpoint as the example below:
+
+```json
+{
+  "selectedAddresses": [
+    {
+      "addressId": "5139683010323",
+      "addressType": "search",
+      "city": null,
+      "complement": null,
+      "country": "BRA",
+      "neighborhood": null,
+      "number": null,
+      "postalCode": "08391705",
+      "receiverName": "Jane Doe",
+      "reference": null,
+      "street": null,
+      "addressQuery": "",
+      "isDisposable": null
+    }
+  ],
+  "expectedOrderFormSections": [
+    "items",
+    "totalizers",
+    "clientProfileData",
+    "shippingData",
+    "paymentData",
+    "sellers",
+    "messages",
+    "marketingData",
+    "clientPreferencesData",
+    "storePreferencesData",
+    "giftRegistryData",
+    "ratesAndBenefitsData",
+    "openTextField",
+    "commercialConditionData",
+    "customData"
+  ]
+}
+```
+
+Once configured, the `orderForm` will retrieve the variable `shippingEstimateDate` with the shipping estimate date being displayed, instead of only the shipping estimate in business days.
```

Comment:
```suggestion
Once configured, the `orderForm` will retrieve the variable `shippingEstimateDate`, which will display the shipping estimate date instead of only the shipping estimate in business days.
```